backbone/README.md

```diff
@@ -2,23 +2,369 @@
 
 Guidelines used by Eventbrite to provide consistency and prevent errors in JavaScript code written for [Backbone.js](backbonejs.org) and [Marionette.js](marionettejs.com).
 
+Backbone and Marionette come with a rich API and also functions provided by [underscore](http://underscorejs.org/) (`_`) and [jquery]((http://api.jquery.com/category/version/1.7/)) (`$`). Although good and fast to use, these utilities can be hard to navigate or even challenging when building large-scale applications. Many times midway through development, we find that were used the tools incorrectly and have to change course, resulting in Frankenstein code. This guide will attempt to ease some of these problems.
+
 ## Table of Contents
 
-1. [What is Backbone.js?](#what-is-backbonejs)
-2. [What is Marionette.js?](#what-is-marionettejs)
+0. [Backbone.js](#backbonejs)
+0. [Marionette.js](#marionettejs)
+0. [Additional plugins](#additional-plugins)
+0. [Common terminology](#common-terminology)
+0. [File structure](#file-structure)
+0. [Statics](#statics)
+0. [Styling](#styling)
+0. [Context](#context)
+0. [Function](#good-practices-functions)
```

Comment:
I wonder if we should move this (and below) into a "TODO" section since there isn't any content...